lib/manager/mix/readme.md

```diff
@@ -0,0 +1,126 @@
+## Overview
+
+#### Name of package manager
+
+Mix
+
+#### What language does this support?
+
+Elixir
+
+#### Does that language have other (competing?) package managers?
+
+Mix is main Elixir build tool
+
+## Package File Detection
+
+#### What type of package files and names does it use?
+
+Erlang: rebar.config
```

Comment:
Can you update this file so that it's referring only to Mix, and we do rebar separately?